runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/CredentialTile/CredentialTile.tsx

```diff
@@ -1,38 +1,38 @@
 import { FC } from 'react';
 
-import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
 import FolderOpenIcon from '@mui/icons-material/FolderOpen';
-import { Box, CircularProgress, IconButton, Typography } from '@mui/material';
+import { CircularProgress, Typography } from '@mui/material';
 
 import { useRouter } from 'next/router';
 
+import { PrivilegeType, DEFAULT_COLLECTION_COLOR, FrontCredentialDto, FrontCredentialCollectionDto } from 'runbotics-common';
+
 import If from '#src-app/components/utils/If';
 import useAuth from '#src-app/hooks/useAuth';
 import useTranslations from '#src-app/hooks/useTranslations';
-import { BasicCredentialDto } from '#src-app/views/credentials/Credential/Credential.types';
-import { BasicCredentialsCollectionDto, PrivilegeType } from '#src-app/views/credentials/CredentialsCollection/CredentialsCollection.types';
 import { collectionColors } from '#src-app/views/credentials/CredentialsCollection/EditCredentialsCollection/CollectionColor/CollectionColor.utils';
 
 import { CredentialCard, CredentialCollection } from './CredentialTile.styles';
+import MenuItems from '../CredentialsCollectionTile/MenuItems/MenuItems';
 import Tile from '../Tile';
 
 export interface CredentialTileProps {
-    credential: BasicCredentialDto;
-    collection: BasicCredentialsCollectionDto;
+    credential: FrontCredentialDto;
+    collection: FrontCredentialCollectionDto;
```

Comment:
why not just `CredentialDto` and `CredentialCollectionDto`, if they were used in rb-ui only, why do we need to create them in rb-common?